catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -195,6 +198,15 @@ public void create() {
 		virtualWidthHalf = virtualWidth / 2;
 		virtualHeightHalf = virtualHeight / 2;
 
+		font = new BitmapFont();
+
+		Context context = CatroidApplication.getAppContext();
```

Comment:
this bothers me. i want to get rid of all the CatroidApplication.getAppContext calls, not introduce new ones.
id rather have this resource getting stuff somewhere else, since this seems to be the only point in this whole class that requires a context. one possibility would be that the stageActivity has a axesColor.